task_sdk/tests/execution_time/test_supervisor.py

```diff
@@ -320,30 +314,30 @@ def watched_subprocess(self, mocker):
     @pytest.mark.parametrize(
         ["message", "expected_buffer", "client_attr_path", "method_arg", "mock_response"],
         [
-            pytest.param(
-                GetConnection(conn_id="test_conn"),
-                b'{"conn_id":"test_conn","conn_type":"mysql"}\n',
-                "connections.get",
-                ("test_conn",),
-                ConnectionResult(conn_id="test_conn", conn_type="mysql"),
-                id="get_connection",
-            ),
-            pytest.param(
-                GetVariable(key="test_key"),
-                b'{"key":"test_key","value":"test_value"}\n',
-                "variables.get",
-                ("test_key",),
-                VariableResult(key="test_key", value="test_value"),
-                id="get_variable",
-            ),
-            pytest.param(
-                GetXCom(dag_id="test_dag", run_id="test_run", task_id="test_task", key="test_key"),
-                b'{"key":"test_key","value":"test_value"}\n',
-                "xcoms.get",
-                ("test_dag", "test_run", "test_task", "test_key", -1),
-                XComResult(key="test_key", value="test_value"),
-                id="get_xcom",
-            ),
+            # pytest.param(
+            #     GetConnection(conn_id="test_conn"),
+            #     b'{"conn_id":"test_conn","conn_type":"mysql"}\n',
+            #     "connections.get",
+            #     ("test_conn",),
+            #     ConnectionResult(conn_id="test_conn", conn_type="mysql"),
+            #     id="get_connection",
+            # ),
+            # pytest.param(
+            #     GetVariable(key="test_key"),
+            #     b'{"key":"test_key","value":"test_value"}\n',
+            #     "variables.get",
+            #     ("test_key",),
+            #     VariableResult(key="test_key", value="test_value"),
+            #     id="get_variable",
+            # ),
+            # pytest.param(
+            #     GetXCom(dag_id="test_dag", run_id="test_run", task_id="test_task", key="test_key"),
+            #     b'{"key":"test_key","value":"test_value"}\n',
+            #     "xcoms.get",
+            #     ("test_dag", "test_run", "test_task", "test_key", -1),
+            #     XComResult(key="test_key", value="test_value"),
+            #     id="get_xcom",
+            # ),
```

Comment:
Unrelated?

Reply:
Ah yes, was working on a different branch and this went in :/